ietf-document/draft-0rtt-bdp/draft-kuhn-quic-0rtt-bdp-10.xml

```diff
@@ -370,6 +381,19 @@
                                 estimated parameters would not be
                                 valid.
 				</t>
+				<t>
+                                Relevance of the BB estimation: The
+                                bottleneck bandwidth estimation may
+                                occur just at the end of the slow start.
+                                There are cases where the congestion
+                                window at the end of slow start is
+                                significantly larger than the value on
+                                which the connection converges after a
+                                few rounds. Directly exploiting such
+                                value for the bottleneck bandwith
+                                estimation may be inappropriate.
+ 				</t>
```

Comment:
Careful here. There are errors on the CWIN, pretty much for all of Reno, Cubic, Hystart, BBR. The bandwidth situation is different. BBR never really overestimates the bottleneck bandwidth, it is always "sum of data acknowledged over interval divided by length of interval". Reno and Cubic do not have a concept of bottleneck bandwidth, so developers might be tempted to set it as "CWIN/RTT", and thus import the uncertainty on CWIN into the BB estimate.

Reply:
Does commit https://github.com/NicoKos/QUIC_HIGH_BDP/commit/9dec6d11b1dfc74250882ff50dcb5bd70814bf1f resolves the issue ?